### PR TITLE
Edit translation message on account page.

### PR DESCRIPTION
**Pull request by visortelle**
Change the translation message from "Spoken languages" to "Spoken language". 
The plural form can confuse a user because select input allows selecting only a single value.

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @visortelle! I've created [OSPR-5300](https://openedx.atlassian.net/browse/OSPR-5300) to keep track of it in JIRA, where we prioritize reviews. Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.


:warning: We can't start reviewing your pull request until you've submitted a     [signed contributor agreement](https://open.edx.org/wp-content/uploads/2019/01/individual-contributor-agreement.pdf)     or indicated your institutional affiliation. Please see the [CONTRIBUTING](https://github.com/edx/edx-platform/blob/master/CONTRIBUTING.rst)     file for more information.

**Comment by visortelle** [comment]
I think this PR is ready for review. :)

Screen Shot 2020-12-12 at 5 56 49 PM

**Comment by natabene** [comment]
@visortelle Thank you for joining the Open edX community! We have added your user to our records and can now review your code going forward.

**Comment by natabene** [comment]
@nedbat Not sure how to kick Travis here, any ideas?

File: src/account-settings/AccountSettingsPage.messages.jsx [hunk 1]
```diff
@@ -279,7 +279,7 @@ const messages = defineMessages({
   },
   'account.settings.field.language.proficiencies': {
     id: 'account.settings.field.language.proficiencies',
-    defaultMessage: 'Spoken languages',
+    defaultMessage: 'Spoken language',
     description: 'Label for account settings spoken languages field.',
```
Comment (sarina) [inline src/account-settings/AccountSettingsPage.messages.jsx:283]: Should this be `spoken language` here as well?

Comment (visortelle) [reply src/account-settings/AccountSettingsPage.messages.jsx]: Fixed.
@sarina take a look at https://github.com/edx/frontend-app-account/pull/361/commits/1a0fc71923b48ab63342b7d7f999e8dc8fcc88ca

**Comment by nedbat** [comment]
I don't know how to get Travis to go.  I thought the branch build would be listed at travis.com, but I can't find it.  The author can force it by push a new commit (even an empty one) to the branch.

**Comment by visortelle** [comment]
@nedbat , I already tried.

Links:

https://github.com/visortelle/frontend-app-account/runs/1573088662

https://travis-ci.com/github/visortelle/frontend-app-account/builds/209470351

Trying once more with a new empty commit. Will see. :)

https://github.com/visortelle/frontend-app-account/runs/1579629083

**Comment by natabene** [comment]
A random question - would closing and re-opening the PR help?

**Comment by davidjoy** [comment]
The JSON files in src/i18n/messages shouldn't be changed - they're automatically generated from Transifex and will be overridden as soon as the job runs.  Only the messages file needs to be updated.

Updating the PR to fix this will also cause Travis to re-run, I hope. 😁

**Comment by visortelle** [comment]
@davidjoy I reverted back changes in `src/i18n/messages/*` files.

CI checks are passed now. 👍

**Comment by natabene** [comment]
@davidjoy We still have 13 minutes to merge before code freeze :)

**Review by davidjoy — APPROVED** [review]

**Comment by edx-webhook** [comment]
@visortelle 🎉 Your pull request was merged!

Please take a moment to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=edx/frontend-app-account&entry.867055334=https://github.com/edx/frontend-app-account/pull/361&entry.1484655318=https://github.com/visortelle&entry.752974735=2020-12-11+13:13&entry.1917517419=2020-12-22+18:36&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**